access-control-app/src/cmr/access_control/services/acl_validation.clj

```diff
@@ -214,21 +207,21 @@
         ungrantable-permissions (remove (set grantable-permissions) permissions-requested)]
     (when (and (seq ungrantable-permissions) (seq (set grantable-permissions)))
       {key-path [(format "[%s] ACL cannot have [%s] permission for target [%s], only [%s] are grantable"
-                         (name identity-type) (clojure.string/join ", " ungrantable-permissions)
-                         target (clojure.string/join ", " grantable-permissions))]})))
+                         (name identity-type) (string/join ", " ungrantable-permissions)
+                         target (string/join ", " grantable-permissions))]})))
 
 (defn- make-acl-validations
   "Returns a sequence of validations closed over the given context for validating ACL records.
    Takes action flag (:create or :update) to do different valiations based on whether creating or updating acl concept"
-  [context acl action]
+  [context acl]
   [#(validate-provider-exists context %1 %2)
-   {:catalog-item-identity (v/when-present (make-catalog-item-identity-validations context acl action))
+   {:catalog-item-identity (v/when-present (make-catalog-item-identity-validations context acl))
     :single-instance-identity (v/when-present (make-single-instance-identity-validations context))
-    :group-permissions (v/every (make-group-permissions-validation context acl))}
+    :group-permissions (v/every {:group-id (v/when-present group-id-validation)})}
```

Comment:
Why are we replacing this func?

Reply:
I deleted the cmr.access-control.services.acl-validation function above and replaced it with {:group-id (v/when-present group-id-validation)} because that is all that the function did.  It did not use the passed in context nor acl parameters.